EIPS/eip-5988.md

```diff
@@ -0,0 +1,163 @@
+---
+eip: 5988
+title: Add Poseidon hash function precompile
+description: Add a precompiled contract which implements the hash function used in the Poseidon cryptographic hashing algorithm
+author: Abdelhamid Bakhta (@abdelhamidbakhta), Eli Ben Sasson (@Elistark), Avihu Levy (@avihu28), David Levit Gurevich (@DavidLevitGurevich)
+discussions-to: https://ethereum-magicians.org/t/eip-5988-add-poseidon-hash-function-precompile/11772
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-11-15
+---
+
+## Abstract
+
+This EIP introduces a new precompiled contract which implements the hash function used in the Poseidon cryptographic hashing algorithm, for the purpose of allowing interoperability between the EVM and ZK / Validity rollups, as well as introducing more flexible cryptographic hash primitives to the EVM.
+
+## Motivation
+
+[Poseidon](https://eprint.iacr.org/2019/458.pdf) is an arithmetic hash function that is designed to be efficient for Zero-Knowledge Proof Systems.
```

Comment:
We are still not allowing external links, but we are very close to adding support for a few cases - namely papers with DOIs, such as this. Please follow the progress of #5874.

Reply:
Until then, the links either need to be removed or this PR should remain open.

Reply:
Sounds good, I will move everything relevant to assets folder